fastavro/_write_py.py

```diff
@@ -13,18 +13,16 @@
 import time
 import uuid
 from binascii import crc32
-from collections import Iterable, Mapping
 from os import urandom, SEEK_SET
 from struct import pack
 from zlib import compress
-import numbers
 
+from fastavro._validate_py import validate
```

Comment:
I think this can be `from .validate import validate` to match how the others are done.

Reply:
I believe this was from when I was working on the Python version only.